ui/app/src/components/DialogHeader.tsx

```diff
@@ -58,17 +61,21 @@ export interface DialogTitleProps {
   }
   subtitle?: string;
   icon?: any;
+  backIcon?: any;
 
-  onClose(): void;
+  onClose?(): void;
+  onBack?(): void;
 }
 
 export default function DialogHeader(props: DialogTitleProps) {
   const classes = dialogTitleStyles({});
   const {
     onClose,
+    onBack,
     title,
     subtitle,
     icon: Icon = CloseIcon,
```

Comment:
This prop needs to be renamed (closeIcon)

Reply:
this rename is complicated, because this component is used on everyDialog